### PR TITLE
Power VS: use the new name field for the dhcp nw

### PR DESCRIPTION
**Pull request by clnperez**
this will make the destroy logic more robust, as currently we have no way of knowing which networks belonged to which clusters when destroy errors occurred. the current logic is to look at the nw attached to a cluster vm and delete it, but if the cluster vms had been destroyed already, there would be an unused nw left behind.

Signed-off-by: Christy Norman <christy@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by clnperez** [comment]
@rna-afk can you review? i think all the tests pass now

**Review by r4f4 — COMMENTED** [review]
/lgtm

**Comment by clnperez** [comment]
the `approve` label isn't showing up so this one's not getting merged. @rna-afk do we need someone else's approval?

**Comment by rna-afk** [comment]
Well that's weird. Let me try again.

/approve
